server.js

```diff
@@ -201,7 +211,9 @@ function render(req, res) {
 					});
 
 				} catch (error) {
-					console.info("Screenshot", error)
+					let e = "Screenshot " + error;
+					console.info(e);
```

Comment:
```suggestion
					let e = "Screenshot " + error.toString();
					console.error(e);
```
I am not sure if the error object prints out pretty if concat it. Maybe it does.

And then we can console log it as **error** and not **info**.